.github/workflows/ci.yaml

```diff
@@ -0,0 +1,61 @@
+# yamllint disable rule:line-length
+# yamllint disable rule:braces
+
+name: CI
+
+on:
+  pull_request:
+  push:
+    branches:
+      - "master"
+
+jobs:
+  phpunit:
```

Comment:
Why nod add the Coding Standards before the `phpunit` job, in the same workflow?